DnDTests/Services/UtilsServiceTest.cs

```diff
@@ -1,13 +1,18 @@
 ﻿using Enums;
+using Exceptions;
 using Interfaces;
 using Models;
+using Newtonsoft.Json.Linq;
 using Services;
+using Strategies;
 
 namespace DnDTests.Services
 {
     internal class UtilsServiceTest
     {
         private IUtilsService _service;
+        private readonly string _invalidAttributeError = "Attribute must be STR, DEX, CON, INT, WIS or CHA";
+
```

Comment:
i've already read this somewhere ( i know where ;) ) do you really need this here?

Reply:
This text has been removed, but was used to check if the expected error message is equal to the one set in Constants.cs.